### DISCUSSION TITLE
Chromecast streaming in real time

### DISCUSSION THREAD

**Discussion by christianpadovano** [kokorin/Jaffree#222] [Q&A]
Hi ,  it should be possible with this library streaming an mkv file to a Chromecast device in real time ?
Thanks

**Comment by kokorin** [comment]
Hello,

Could you be more specific? How would you cast with ffmpeg itself? What codecs are used in MKV? 

After some googling I found [this snippet](https://gist.github.com/pranavraja/f4a33176e902dbb1b504c21f4e6c972b). So looks like it theoretically possible. 

I don't have a chromecast, so I can't even experiment with it.

**Comment by christianpadovano** [comment]
I need something that can redirect the ffmpeg output stream to chromecast. Chromecast can handle h264, mp4, aac, ac3 passthrought.

I asked u because i ve seen project written in node.js and python using fmpeg to do this job.

Thank u for the help


Scarica Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by kokorin** [comment]
It would be great if you can provide ffmpeg command line the tools you mentioned use.

**Comment by christianpadovano** [comment]
If i will get something usefull i will give you all the informations

Scarica Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
